prospector/commit_processor/test_feature_extractor.py

```diff
@@ -103,7 +111,9 @@ def test_extract_changes_relevant_path():
     advisory_record = AdvisoryRecord(
         vulnerability_id="test_advisory_record", paths=[path_1, path_2]
     )
-    assert extract_changes_relevant_path(commit, advisory_record)
+    assert extract_changed_relevant_paths(commit, advisory_record) == {
+        path_1,
```

Comment:
This should be tested also when the "path" element from the Adv. Record is only a 'piece' of the path in the commit.

Reply:
I fixed it.